statsmodels/tsa/statespace/tools.py

```diff
@@ -1617,7 +1619,7 @@ def copy_missing_matrix(A, B, missing, missing_rows=False, missing_cols=False,
     try:
         if not A.is_f_contig():
             raise ValueError()
-    except:
+    except (AttributeError, ValueError):
```

Comment:
@ChadFulton I'm fairly confident this is what should be caught, but this merits another pair of eyes.